src/components/App/index.tsx

```diff
@@ -1,11 +1,23 @@
-import React from 'react';
+import { logout } from '../../authentication';
+import { EgeriaLogin } from '../Login';
 
 import './index.scss';
 
 export function App() {
+
+  console.log(logout);
   return <>
-    <div className="container">
+    <div>
       <h1>Egeria UI Core</h1>
     </div>
+
+    <div>
+      <h1>Login</h1>
+
+      <EgeriaLogin loginCallback={() => {
+                     window.location.href = `http://localhost:3001/`;
```

Comment:
these urls should be extracted to some other place

Reply:
this is for local development only, it's a component sample